src/tests/mocks/erc721_mocks.cairo

```diff
@@ -162,17 +162,20 @@ mod CamelERC721Mock {
 
     /// The following external methods are included because they are case-agnostic
     /// and this contract should not embed the snake_case impl.
+    #[abi(per_item)]
```

Comment:
Is now required to do it like this? Why do we prefer this instead of the previous version?

Reply:
I'm planning ahead with the nightly version from Scarb. It appears we will get this warning (among others) with the next release: `The 'external' attribute on impls is deprecated. Use 'abi(per_item)' or 'abi(embed_v0)'.`

Reply:
Make sense, but why not using embed instead of per item then? Is similar to what we already have right?

Reply:
The compiler will only let you embed an impl of an interface. We're generating the trait in this particular mock, so I believe this is our best option (it's for case-agnostic methods from ERC721 and ERC721metadata)

Reply:
The per_item annotation is super cool though because it allows function-level annotation within the impl...which reminds me, we can add constructors in the generated impls!